providers/os/connection/docker/docker_file_connection.go

```diff
@@ -86,7 +91,9 @@ func NewDockerfileConnection(id uint32, conf *inventory.Config, asset *inventory
 
 	conn := &DockerfileConnection{
 		LocalConnection: localConn,
-		Filename:        filename,
+		// here we must use the absolute path of the Dockerfile so
+		// that we find the file downstream
+		Filename: absSrc,
```

Comment:
I know @preslavgerchev spent some time looking into the file paths for dockerfiles. He may be a better option to evaluate this PR